python/paddle/fluid/tests/unittests/mkldnn/test_reduce_mkldnn_op.py

```diff
@@ -100,7 +111,10 @@ def setUp(self):
         self.outputs = {'Out': self.inputs['X'].sum()}
 
 
-@OpTestTool.skip_if_not_cpu()
+@OpTestTool.skip_if(
+    True,
+    reason="According to Paddle API, None dim means reduce all instead of copy, so just skip this test to avoid potential failure",
+)
```

Comment:
can you explain more?

Reply:
According to the doc: https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/fluid/layers/nn/reduce_sum_en.html

When dim attr is empty, reduce_sum op should reduce the input tensor to a scalar, instead of just copying the input to output without shape changing. I feel this case is an invalid case, so jsut skip it.

Reply:
okay.